ci/deploy_app.yaml

```diff
@@ -0,0 +1,38 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: theorf/google-cloud-sdk-helm
+inputs:
+  - name: eq-questionnaire-runner-repo
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: '/root/gcloud-service-key.json'
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  DOCKER_REGISTRY: 'eu.gcr.io/census-eq-ci'
+  IMAGE_TAG: 'latest'
+  REQUESTED_CPU_PER_POD: 3
+  ROLLING_UPDATE_MAX_UNAVAILABLE: 1
+  ROLLING_UPDATE_MAX_SURGE: 1
+  MIN_REPLICAS: 3
+  MAX_REPLICAS: 10
+  TARGET_CPU_UTILIZATION_PERCENTAGE: 50
+  COOKIE_SETTINGS_URL: '#'
```

Comment:
These defaults shouldn't be set here. Anywhere we default values we use production settings so there is less risk of deploying non-production settings to production. As we also wouldn't want someone deploying this task to their environment to accidentally use the production settings (over scaled), these should be left blank.